src/components/Notice/Notice.jsx

```diff
@@ -53,30 +53,31 @@ const CloseButton = styled.button`
   position: absolute;
   right: 16px;
   &:hover {
-    color: ${props => colorMap[props.type].color};
+    color: ${(props) => colorMap[props.type].color};
     opacity: 1;
     cursor: pointer;
   }
 `;
 
-function Notice({ children, dismiss, type, className }) {
-  const { AnimationWrapper, dismiss:dismissAnimationWrapper, animationProps } = useAnimation({
+function Notice({ children, dismiss, type, className, disableAnimation }) {
+  const {
+    AnimationWrapper,
+    dismiss: dismissAnimationWrapper,
+    animationProps,
+  } = useAnimation({
     justify: 'flex-end',
     stageInAnimation: stageInRight,
     stageOutAnimation: fadeOut,
     onDismiss: dismiss,
-  })
+  });
 
   return (
-    <AnimationWrapper {...animationProps}>
+    <AnimationWrapper {...animationProps} disableAnimation={disableAnimation}>
```

Comment:
Would it work to just not wrap it in the `AnimationWrapper` at all if `disableAnimation` is true? 🤔